tests/integration/test_dbapi_integration.py

```diff
@@ -1343,23 +1343,13 @@ def test_cancel_query(trino_connection):
     cur.fetchone()
     cur.cancel()  # would raise an exception if cancel fails
 
-    cur = trino_connection.cursor()
-    with pytest.raises(Exception) as cancel_error:
-        cur.cancel()
-    assert "Cancel query failed; no running query" in str(cancel_error.value)
```

Comment:
can you adjust the test to instead make sure this next cancel doesn't fail with exception?

Reply:
same below

Reply:
done